.NET/Microsoft.Recognizer.Text.TimeZone/Constants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Recognizers.Text.TimeZone
+{
+    class Constants
+    {
+        public const string SYS_TIMEZONE_UTC = "builtin.timezone.utc";
+        public const string SYS_TIMEZONE = "builtin.timezone";
```

Comment:
What's the purpose of this constant? To represent that resolution is not sure of what timezone it is?